portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,51 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="content">
+		<div id="white-box">
+			<h1>Caroline's Portfolio</h1>
+			<div id="blurb">
+				<p>I'm Caroline and this is my portfolio! A bit about me: I'm currtenly a rising Junior at Cornell's
+					College
+					of
+					Engineering, majoring in Computer Science. On campus, I sing in the University Chorus and I also
+					work as
+					a
+					TA for CS 2800: Discrete Math, one of my favorite courses I've taken at Cornell. My interests
+					outside CS
+					include Music and Linguistics, but I've known I wanted to study Computer Science ever since I
+					received
+					my
+					first programmable calculator in middle school! Since beginning my degree, I've taken many
+					fascinating
+					classes in many areas of the field, including Natural Language Processing, Artificial Intelligence,
+					Smartphone Computing Tech, Functional Programming, Data Structures, and more! I've loved them all,
+					but I
+					have a special place in my heart for theory classes like Algorithms. It's hard to say whether I have
+					a
+					favorite field within CS, since I'm still exploring and learning about new career paths and options,
+					but
+					I've had a great time so far working with robotics, data management, machine learning, and software
+					engineering, and I can't wait to see what more I'll learn during this internship! </p>
+			</div>
+		</div>
+		<p> Come check me out on <a href="https://www.linkedin.com/in/caroline-lui-877638174/"
+				target="_blank">LinkedIn</a> and <a href="https://github.com/cel243" target="_blank">GitHub</a>!
+		</p>
+		<p>Below is my dog, Penny!</p>
+		<div id=penny-image>
+			<a href="images/Penny-1.JPG"><img src="images/Penny-1.JPG"/></a>
+
```

Comment:
nit: could you please get rid of this extra new line?

Reply:
new line is gone now!